src/Context/ContextApi.php

```diff
@@ -55,6 +55,18 @@ protected function processBodyOptions(array $requestData = []) {
     }
 
     /**
+     * Returns X-SL-Context-Source header.
+     *
+     * @return string
+     */
+    public function getXSLContextSourceHeader() {
```

Comment:
Why it's `public`? I don't expect that someone will use it outside of this class